src/pages/data-stack/sql-editor.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import ReaderView from 'components/ReaderView'
+import { customerDataInfrastructureNav } from '../../hooks/useCustomerDataInfrastructureNavigation'
+import { TreeMenu } from 'components/TreeMenu'
+import SEO from 'components/seo'
+import Link from 'components/Link'
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import OSTable from 'components/OSTable'
+
+const LeftSidebarContent = () => {
+    return <TreeMenu items={customerDataInfrastructureNav.children} />
+}
+
+type SQLEditorFeature = {
+    title: string
+    description: string
+    comingSoon?: boolean
+}
+
+const sqlEditorFeatures: SQLEditorFeature[] = [
+    {
+        title: 'Direct data access',
+        description:
+            'Query all your data in PostHog, including PostHog-specific tables (events, persons) and synced warehouse sources',
+    },
+    {
+        title: 'HogQL support',
+        description:
+            "Uses PostHog's SQL variant with simplified property access (dot notation like properties.$current_url), null handling, and ClickHouse functions",
+    },
+    {
+        title: 'Autocomplete',
+        description: 'Intelligent code completion for tables, columns, and SQL keywords to speed up query writing',
+    },
+    {
+        title: 'Schema browser',
+        description:
+            'View and search the schema of all available sources, PostHog tables, and saved views directly in the editor',
+    },
+    {
+        title: 'Save as views',
+        description: 'Save queries as views accessible from the data warehouse tab, visible to all project users',
+    },
+    {
+        title: 'Draft queries',
+        description: 'Save draft queries for later without cluttering the saved views list',
+    },
+    {
+        title: 'DuckDB syntax support',
+        description: 'Full support for DuckDB SQL syntax and functions when connected to a managed DuckDB warehouse',
+        comingSoon: true,
+    },
+    {
+        title: 'Add queries to notebooks',
+        description: 'Use PostHog notebooks to collect warehouse info, research topics, or just as a scratch pad',
+    },
+    {
+        title: 'Write SQL without knowing SQL',
+        description: 'Nobody likes writing SQL, except for PostHog AI',
+    },
+    {
+        title: 'Simplified syntax',
```

Comment:
most of these points won't be compatible with the warehouse duckdb syntax as that's just pure duckdb